tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -1160,28 +1162,67 @@ export class WebGPUBackend extends KernelBackend {
 
   fromPixels(
       pixels: backend_util.PixelData|ImageData|HTMLImageElement|
-      HTMLCanvasElement|HTMLVideoElement,
+      HTMLCanvasElement|HTMLVideoElement|ImageBitmap,
       numChannels: number): Tensor3D {
     if (pixels == null) {
       throw new Error(
           'pixels passed to tf.browser.fromPixels() can not be null');
     }
 
+    // TODO(tfjs-optimization): These has chance to be cached and do not update
+    // buffer always.
```

Comment:
Could you explain this comment a little bit more?

Reply:
Thanks for reviewing!
This means we will generate multiple shaders with fixed numbers (like current workGroupSize)

Reply:
Done.